### PR TITLE
Use default arguments removed by mypy

### PR DESCRIPTION
**Pull request by madera-wizeline**
## What does this PR do? (required)
* Re-adds the default arguments removed by PR https://github.com/wizeline/relayer/pull/41. 

## Are there any relevant tickets, PRs, or tech specs?
* Unblocks hubble digest PR (Pending)
 
## Screenshots
screen shot 2018-06-20 at 12 27 28 pm

screen shot 2018-06-20 at 2 31 48 pm

### CODE REVIEW

File: relayer/flask/__init__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 class FlaskRelayer(object):
 
-    def __init__(self, app: Flask, logging_topic: str, kafka_hosts: str = None, **kwargs: str) -> None:
+    def __init__(self, app: Flask = None, logging_topic: str = None, kafka_hosts: str = None, **kwargs: str) -> None:
```
Comment (CesarLanderos) [inline relayer/flask/__init__.py]: isn't `kwargs` a dict?

Comment (madera-wizeline) [reply relayer/flask/__init__.py]: yup, lemme fix that also

Comment (CesarLanderos) [reply relayer/flask/__init__.py]: 👍

Comment (madera-wizeline) [reply relayer/flask/__init__.py]: For some reason changing it to `Dict` causes it to fail 😢 
I also had to change the init_app argument to `Any` for it to pass the test.

Comment (CesarLanderos) [reply relayer/flask/__init__.py]: did you change both kwargs to dict here and in `init_app`?

Comment (CesarLanderos) [reply relayer/flask/__init__.py]: and everywhere... there is a lot of places where kwargs is typed as a `str` :s

Comment (madera-wizeline) [reply relayer/flask/__init__.py]: let me try a full refactor. If it takes more than 30 min then its probably not worth it. :loading:

Comment (madera-wizeline) [reply relayer/flask/__init__.py]: nope it will take forever 🤒 😭

Comment (madera-wizeline) [reply relayer/flask/__init__.py]: Opened [this issue](https://github.com/wizeline/relayer/issues/46) to dedicate more time to debugging this later on.

Comment (pabloariasperez) [reply relayer/flask/__init__.py]: Remember that `*` and `**` is for unpacking sequences and dicts respectively. So... the typing is not a `Dict` but a description of the contained objects. So, when you type as `str` it is expecting that all elements are `str` (http://mypy.readthedocs.io/en/latest/cheat_sheet_py3.html)
```py
# This makes each positional arg and each keyword arg a "str"
def call(self, *args: str, **kwargs: str) -> str:
    request = make_request(*args, **kwargs)
    return self.do_api_query(request)
```

So, the approaches to proceed is:
1. Be explicit about which values are you receiving
2. Use `override` decorator
3. Use `Any` for describing the content

**Pull request merged**